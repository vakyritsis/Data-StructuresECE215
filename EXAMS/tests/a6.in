406 277 146  99  22  6  12  88  45  97 112 111 130 136 171 155 154 161 158 168 244 199 183 261 247 275 272 311 284 278 304 381 348 338 318 346 372 350 375 389 384 395 392 397 724 558 519 440 411 417 483 442 441 490 541 538 527 540 547 637 607 596 628 615 694 664 638 668 708 699 716 887 800 769 728 752 778 777 788 844 836 868 846 926 898 897 907 903 938 931 933 981 942 -1
-a
