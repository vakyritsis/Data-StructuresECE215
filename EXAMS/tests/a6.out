AVL NOT OK in node 311
